wbst-thry/src/app/graphics/lnr-graphics/playground.ts

```diff
@@ -47,7 +74,7 @@ export class Playground {
 
             tick++;
         });
-
-        return scene;
+        moon.actionManager = new Babylon.ActionManager(scene);
+        MeshTriggers.highlightOnHover(moon, highlightLayer, Babylon.Color3.Purple());
```

Comment:
Purple for now, but I'm thinking we can pick RGB values based on the planet's color, something that works with it to contrast and highlight